modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/types.ts

```diff
@@ -17,6 +17,12 @@ export type TAction =
 				selectedObjectDefinitionName: string;
 			};
 			type: TYPES.SET_SELECTED_NODE;
+	  }
+	| {
+			payload: {
+				newElements: any;
+			};
+			type: TYPES.SET_ELEMENTS;
```

Comment:
@igor-franca please, sort this in alphabetical order.

Reply:
Just saw this was done in another commit. In future cases, please squash the changes to facilitate the review.